lib/assets/javascripts/cartodb/table/overlays/map_overlays.js

```diff
@@ -603,11 +658,13 @@ cdb.admin.MapOverlays = cdb.core.View.extend({
 
   _setupCartoDBLogo: function(type, data) {
 
-    var overlay = this.cartodb_logo = new cdb.admin.overlays[type]({
+    var overlay = this.logo = new cdb.admin.overlays[type]({
```

Comment:
I see a line in `clean_overlays` nulling out `this.logo` but it's not a switch from `this.cartodb_logo` -- just want to make sure nothing was missed here.